data/tribes/initialization/empire/units.lua

```diff
@@ -1147,24 +1155,43 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_warehouse",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire warehouse: Warehouse
+            lore = pgettext("empire_building", "‘Bis dat, qui cito dat, nil dat, qui munera tardat.’ (Who gives promptly, gives twice, who gives late, gives nothing.)"),
+            -- TRANSLATORS: Lore author helptext for an empire warehouse: Warehouse
+            lore_author = pgettext("proverb written over the doors of a warehouse"),
             -- TRANSLATORS: Purpose helptext for an empire warehouse: Warehouse
             purpose = pgettext("empire_building", "Your workers and soldiers will find shelter here. Also stores your wares and tools.")
          }
       },
       {
          name = "empire_port",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire warehouse: Port
+            lore = pgettext("empire_building", "‘Navigare necesse est.’ (To sail is necessary.)"),
+            -- TRANSLATORS: Lore author helptext for an empire warehouse: Port
+            lore_author = pgettext("Admiral Litus Marius Exsiccatus"),
             -- TRANSLATORS: Purpose helptext for an empire warehouse: Port
             purpose = pgettext("empire_building", "Serves as a base for overseas colonization and trade. Also stores your soldiers, wares and tools."),
             -- TRANSLATORS: Note helptext for an empire warehouse: Port
             note = pgettext("empire_building", "Similar to the Headquarters a Port can be attacked and destroyed by an enemy. It is recommendable to send soldiers to defend it.")
          }
       },
 
+--[[ lore template
+            -- TRANSLATORS: Lore helptext for an empire production site:
+            lore = pgettext("empire_building", "‘’ ()"),
+            -- TRANSLATORS: Lore author helptext for an empire production site:
+            lore_author = pgettext("???"),
+]]--
+
       -- Small
       {
          name = "empire_quarry",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Quarry
+            lore = pgettext("empire_building", "‘Lapides at ædificium.’ (Stones for Buildings.)"),
+            -- TRANSLATORS: Lore author helptext for an empire production site: Quarry
+            lore_author = pgettext("Handbook of the imperial econony"),
```

Comment:
```suggestion
            lore_author = pgettext("Handbook of the imperial economy"),
```

Reply:
I like the idea of an empire economy handbook. but as the barbarian equivalent is writtten on ox hides mab we could cite the exact stone tablet of the complete handbook. 
this would make a pretty good source for different cites probably

Reply:
Well, I've introduced the Encyclopædia Imperica for this purpose…

Also stone tablets are so 1200 years before the foundation of Fremil. The Empire is not some kind of backwards Sumer Kingdom from ancient times. :) (Romans used papyrus and parchment / vellum – which is just cowhide in a fancier form…)